mm2src/coins_activation/src/platform_coin_with_tokens.rs

```diff
@@ -165,6 +166,8 @@ pub trait PlatformWithTokensActivationOps: Into<MmCoinEnum> {
         storage: impl TxHistoryStorage,
         initial_balance: Option<BigDecimal>,
     );
+
+    fn handle_balance_streaming(&self, config: &EventStreamConfiguration);
```

Comment:
For now, it enables balance streaming only for coins implementing `PlatformWithTokensActivationOps`. We will need it for `InitStandaloneCoinActivationOps`, `InitL2ActivationOps` and maybe coins activated by legacy RPC methods (`enable/electrum`). I think there should be a way to do it for all coins, not depending on the activation method, but it requires additional research on the possible design. So it seems a topic for future sprints.

Reply:
COIN_BALANCE is introduced for tendermint protocol, therefore the current design fully-fits it's requirement. For sure we will support all coins, but as you mentioned it's  a topic for future sprints.